docs/index.md

```diff
@@ -162,8 +162,45 @@ When an issuer generates a new key to sign Health Cards, the public key SHALL be
+
+### Revocation
+
+Individual Health Cards MAY be revoked using a revocation identifier property `rid` encoded in the `vc` claim of the JWT. This should be a short identifier, meaningless to the verifiers; the only constraint is that the identifier SHALL use the base64url alphabet (but doesn’t need to be base64url encoded) and be no longer than 24 characters. Issuers MAY use application-specific user identifiers for this purpose, but since these could be publicly listed in revocation lists, issuers SHOULD use a one-way transformation of the data combined with enough entropy to prevent reversal. It is RECOMMENDED to use the base64url encoding of the first 64 bits of the output of HMAC-SHA-256 (as specified in [RFC 4868](https://tools.ietf.org/html/rfc4868)) on the user identifier using a 256-bit random secret key concatenated with the `<<kid>>`; i.e., `rid = base64url(hmac-sha-256(secret_key || <<kid>>, user_id)[1..64]).
+
+To enable per-card revocation, the issuer creates, for each of its keys, a JSON Card Revocation List (CRL) file with the following content:
+```json
+{
+"kid": "<<kid>>",
+"method": "rid",
+"ctr": "<<ctr>>",
+"rids": [...]
+}
+```
+
+where
+- `"<<kid>>"` is the ID of the corresponding issuer key,
+- `"rid"` identifies the revocation method specified in this framework; legacy cards can use different methods specified in external revocation profiles,
+- `"<<ctr>>"` is counter indicating how many times this file has been updated; initial value is 1,
+-  `rids` is an array of revoked cards' identifiers `rid` (using the base64url alphabet) combined with optional timestamps.
+
+To revoke a Health Card issued under the key `"<<kid>>"`, an issuer adds its revocation identifier to the `rids` array of the corresponding `<<kid>>`'s revocation file. Since an issuer might want to invalidate a series of Health Cards associated with the user up to a certain time, the `rid` might be followed by a separator `.` a timestamp (encoded as the number of seconds from 1970-01-01T00:00:00Z UTC, as specified by RFC 7519). After updating the `rids` array (with one or more items), the `<<ctr>>` is incremented.
+
+As an example, the `rids` array `[AQPCj4wwk6Mt, lHKzqFUMjhs.1636977600]` marks as revoked any Health Cards with `rid` equal to `AQPCj4wwk6Mt` and Health Cards with `rid` equal to `lHKzqFUMjhs` issued before November 15, 2021 12:00:00 PM GMT.
+
+The per-key revocation file is made available at `https://"<<Issuer URL>>"/.well-known/crl/kid/"<<kid>>".json`, where
+- `"<<Issuer URL>>"` is the issuer URL listed in the Health Card,
+- `"<<kid>>"` is the key ID with which the Health Card was signed.
+
+Issuers supporting this revocation method SHALL include in their published JWK set, for each key, a `crl-version` field encoding the update counter "<<ctr>>" for the corresponding revocation file.
+
+If the `crl-version` is present in the Issuer's JWK for key `<<kid>>`, Verifiers SHALL
```

Comment:
Can we call this `crlVersion`? Would like to follow a common JSON naming schemes compatible with JS variable names...

Reply:
Done.